packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -410,7 +425,7 @@ function runTest<TChannelFactory extends IChannelFactory, TOperation extends Bas
 			const channel: ReturnType<typeof model.factory.create> = model.factory.create(
 				dataStoreRuntime,
 				// Use friendly IDs for a reasonable number of DDSes for easier debugging.
-				index < 26 ? String.fromCharCode(index + 65) : random.uuid4(),
+				index < 26 ? String.fromCodePoint(index + 65) : random.uuid4(),
```

Comment:
wouldn't a uuidv5(index) or just to stringing the index work here? This seems strange, but I think your change is fine.

Reply:
the linter specifically complained about using `fromCharCode` and wanted `fromCodePoint`. I'm indifferent for this usage (they return the same thing on the range 65 - 89) so I just satisfied the linter.

using uuidv5 would also work fine, but so would not supplying an ID at all. the intent here is to make failure json more human-readable: it's a lot more easy to reason about fuzz testing 3 clients A, B, and C than it is when the clients have guids. if you have more than 26 clients, you probably aren't doing much by-hand minimization or interpretation so we just switch to guids.